go.mod

```diff
@@ -1,26 +1,27 @@
-module github.com/corazawaf/coraza-spoa
+module coraza-spoa
```

Comment:
Wonder about this change, is it necessary? I think it is better to be consisten with the other connectors and use `github.com/corazawaf/coraza-spoa`